composer.json

```diff
@@ -29,13 +35,14 @@
     "require-dev" : {
```

Comment:
I'd rather we don't add dependencies unless absolutely required, so I removed `symfony/http-foundation`